R/theme_nhs.R

```diff
@@ -0,0 +1,58 @@
+#' Add NHS theme to ggplot chart
+#'
+#' This adds a theme to your chart to follow the
+#' [NHS Identity](https://digital.nhs.uk/about-nhs-digital/corporate-information-and-documents/nhs-digital-style-guidelines/how-we-look/charts)
+#'
+#' @importFrom ggplot2 element_text margin element_blank element_line element_rect
+#' @export
+theme_nhs <- function() {
+
+  # Function to help standardise the text
+  nhs_element_text <- function(...) {
+    element_text(family = "Frutiger",
```

Comment:
I think using this font is going to be problematic, as it's not an open source font, so I think we may need to have this as the default if installed, otherwise fall back to default sans serif font (Arial is likewise complicated as it's propriertary). Suggest we replace this with something like
`family = getOption("nhsrtheme.font", "")`
that will default to whatever "" (same as the ggplot default value) if the option hasn't been set